src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.component.js

```diff
@@ -148,9 +149,10 @@ export const WidgetEnrollmentPlain = ({
                             <span className={classes.icon} data-test="widget-enrollment-icon-clock">
                                 <IconClock16 color={colors.grey600} />
                             </span>
-                            {i18n.t('Last updated {{date}}', {
-                                date: moment(fromServerDate(enrollment.updatedAt)).fromNow(),
-                            })}
+                            {i18n.t('Last updated')}
+                            <Tooltip content={(fromServerDate(enrollment.updatedAt).toLocaleString())}>
+                                {moment(fromServerDate(enrollment.updatedAt)).fromNow()}
```

Comment:
After I edit the enrollment, I see the text `Last updated in 2 hours` and the tooltip shows a date that is 2 hours away. The bug is happening in play too. Can you take a look to see if this time zone discrepancy is easy to fix in this PR or if we need to open another bug ticket?  Thanks!
<img width="313" alt="Screenshot 2024-07-02 at 07 42 09" src="https://github.com/dhis2/capture-app/assets/6817073/b24ded31-2f64-486d-98c1-ab897283110c">